pubsub/gcloud/aio/pubsub/publisher_client.py

```diff
@@ -65,6 +65,54 @@ async def _headers(self) -> Dict[str, str]:
     # TODO: implement that various methods from:
     # https://github.com/googleapis/python-pubsub/blob/master/google/cloud/pubsub_v1/gapic/publisher_client.py
 
+    # https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics/list
+    async def list_topics(self, project: str,
+                          query_params: Optional[Dict[str, str]] = None,
+                          *, session: Optional[Session] = None
+                          ) -> Dict[str, Any]:
+        """
+        List topics
+        """
+        url = f'{API_ROOT}/{project}/topics'
+        headers = await self._headers()
+        s = AioSession(session) if session else self.session
+        resp = await s.get(url, headers=headers, params=query_params)
+        result: Dict[str, Any] = await resp.json()
+        return result
+
+    # https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics/create
+    async def create_topic(self, topic: str,
+                           body: Optional[Dict[str, Any]] = None,
+                           *, session: Optional[Session] = None,
+                           timeout: Optional[int] = None
+                           ) -> Dict[str, Any]:
+        """
+        Create topic.
+        """
+        url = f'{API_ROOT}/{topic}'
+        headers = await self._headers()
+        encoded = json.dumps(body or {}).encode()
+        s = AioSession(session) if session else self.session
+        kwargs = {
+            'timeout': timeout,
+        }
+        kwargs = {k: v for k, v in kwargs.items() if v is not None}
```

Comment:
Rather than this kwargs stuff, I think it makes more sense to just pass in the `timeout` default which makes the most sense for `create_topic` and pass it as-is; sounds like `10` is still the right value here. See my comment on Alex's PR for more info!